src/app/communities/communities-edit.component.html

```diff
@@ -6,6 +6,22 @@
         <input matInput id="communities-edit-title" placeholder="Title" formControlName="title" [required]="true">
       </mat-form-field>
     </div>
+    <div fxFlex>
+      <span>Community type:</span>
+      <mat-button-toggle-group formControlName="type" class="communities-edit-type">
+        <mat-button-toggle value="OPENED" class="communities-edit-type-option">Opened</mat-button-toggle>
+        <mat-button-toggle value="CLOSED" class="communities-edit-type-option">Closed</mat-button-toggle>
+      </mat-button-toggle-group>
+      <div *ngIf="communityForm.get('type').value === 'CLOSED'" class="communities-edit-type-hint">
```

Comment:
From my point of view it is better to extract the condition into a method called `isClosedCommunity()`. It is not just a check on `null` or `undefined` and it is quite business-connected to be in a template.

Reply:
was moved to the modal dialog